python/paddle/fluid/tests/unittests/test_static_save_load_bf16.py

```diff
@@ -22,6 +22,7 @@
 from paddle.fluid.optimizer import SGDOptimizer
 from paddle.fluid.tests.unittests.test_imperative_base import new_program_scope
 from paddle.fluid.tests.unittests.test_static_save_load import PtbModel
+from paddle.fluid.tests.unittests.op_test import convert_float_to_uint16
```

Comment:
`convert_float_to_uint16` is probably not needed anymore.

Reply:
You're absolutely right, thank you!